R/t-distribution

```diff
@@ -0,0 +1,270 @@
+# App: t_distribution
```

Comment:
filename should be `app.R` for shiny and Rstudio integrations to work. `.R` extension is also nice...